.github/workflows/check-latest-release.yml

```diff
@@ -92,7 +95,7 @@ jobs:
             search_output=$(gh issue list --search "$title" --label "$label")
 
             GITHUB_WORKFLOW_URL=https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID
-            body="Latest lifecycle release v${{ steps.read-go.outputs.latest-release-version }} triggered CVE(s) from Grype. For further details, see: $GITHUB_WORKFLOW_URL"
+            body="Latest lifecycle release v${{ steps.read-versions.outputs.latest-release-version }} triggered CVE(s) from Grype. For further details, see: $GITHUB_WORKFLOW_URL json: $(cat ${{ steps.scan-image.outputs.json }} | jq '.matches[] | .vulnerability | {id, severity, description}' )"
```

Comment:
@dlion I finally tried these suggestions - 

the jq formatting didn't seem to work:
https://github.com/joe-kimmel-vmw/lifecycle/issues/8

and i guess i don't understand the other suggestion with `join`, .. i just got 
```
The workflow is not valid. .github/workflows/check-latest-release.yml (Line: 84, Col: 14): Unexpected symbol: '|'. Located at position 42 within expression: join('\n', steps.scan-image.outputs.json | fromjson | .matches[].vulnerability | "- | \(.id) | \(.severity) | \(.description) |")
```


IMO most of the value here is provided by having anything at all,  whereas today we don't actually get any useful information from the scan.   I appreciate that nice formatting is nice, but it's a cumbersome debug cycle because I have to hardcode part of the action and then push and run remotely for each iteration,  so i'm not inclined to troubleshooting very much..   I'll leave this open / up overnight in case you want to tell me what i'm doing wrong in the most recent two commits..

Reply:
I think that's closer to being a table but it's not getting escaped for markdown https://github.com/joe-kimmel-vmw/lifecycle/issues/9

i'll try one more time.

Reply:
I added in triple-ticks  so that the table would be interpreted as markdown in the issue but it didn't work - the issue failed to create because i guess somehow then the pipe didn't get interpreted correctly?

https://github.com/joe-kimmel-vmw/lifecycle/actions/runs/5404335134/jobs/9818362129

```
Run #!/usr/bin/env bash
label with name "cve" already exists; use `--force` to update its color and description
/home/runner/work/_temp/3e9ac2b3-28c9-49ba-8d98-e49302e3e0c6.sh: line 14: |: command not found
Error: Process completed with exit code 127.
```

i think at this point honestly printing it as valid json is pretty great anyhow in case somebody wants to paste that output into a script (or an editor/interface that makes it easy to collapse/filter),   and much simpler.  so i'm going to revert to what was previously working.